bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/tracer/BSpan.java

```diff
@@ -158,4 +171,24 @@ public void addTag(String tagKey, String tagValue) {
         }
         return carrierMap;
     }
+
+    public BMap<BString, Object> getBSpanContext() {
+
+        if (bSpanContext == null) {
+            SpanContext spanContext = span.getSpanContext();
+            if (spanContext.isSampled()) {
+                MapType immutableStringMap = TypeCreator.createMapType(PredefinedTypes.TYPE_STRING, true);
+                BMapInitialValueEntry[] values = new BMapInitialValueEntry[]{
+                        new MappingInitialValueEntry.KeyValueEntry(
+                                StringUtils.fromString(TraceConstants.TRACE_ID), spanContext.getTraceId())
+                        , new MappingInitialValueEntry.KeyValueEntry(
```

Comment:
This code formatting is alright?

Reply:
Yes it is commonly used (so that we can remove a entry without affecting the above line).
But I changed it anyway.